azurelinuxagent/common/osutil/default.py

```diff
@@ -1062,8 +1069,11 @@ def route_add(self, net, mask, gateway):
         """
         Add specified route 
         """
-        cmd = "ip route add {0} via {1}".format(net, gateway)
-        return shellutil.run(cmd, chk_err=False)
+        try:
+            cmd = ["ip", "route", "add", net, "via", gateway]
+            return shellutil.run_command(cmd)
```

Comment:
Same comment here; return is not used anywhere.

Reply:
idem